images/tier1/browser.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="18" width="18" viewBox="0 0 18 18">
```

Comment:
Not sure about the terminology of this icon... that doesn't scream "browser" to me but "web" instead.